Formula/aircrack-ng.rb

```diff
@@ -4,7 +4,7 @@ class AircrackNg < Formula
 
   # We can't update this due to linux-only dependencies in >1.1.
   # See https://github.com/Homebrew/homebrew/issues/29450
-  url "https://download.aircrack-ng.org/aircrack-ng-1.1.tar.gz"
+  url "http://download.aircrack-ng.org/aircrack-ng-1.2-rc4.tar.gz"
```

Comment:
Only stable releases are allowed in `url`. If you want a release candidate, add it to `devel` block.

```rb
devel do
  url "http://download.aircrack-ng.org/aircrack-ng-1.2-rc4.tar.gz"
  sha256 "<generated sha256>"
end